src/runtime/auto-prompt-manager.ts

```diff
@@ -727,6 +753,33 @@ export class AutoPromptManager {
     }
   }
 
+  private async handleFrequencyCappingLocalStorage_(
+    analyticsEvent: AnalyticsEvent
+  ): Promise<void> {
+    // Impressions of prompts (for paygated content) do not count toward the
+    // frequency caps. TODO(b/300963305): manually triggered prompts should
+    // also be excluded.
+    if (
+      !INTERVENTION_TO_STORAGE_KEY_MAP.has(analyticsEvent) ||
+      this.pageConfig_.isLocked()
+    ) {
+      return;
+    }
+
+    const storageKey = INTERVENTION_TO_STORAGE_KEY_MAP.get(analyticsEvent)!;
```

Comment:
nit: this can be moved below if statement or inlined in the return line.

Reply:
Good catch.